source/Ship.cpp

```diff
@@ -4213,6 +4213,16 @@ void Ship::SetParent(const shared_ptr<Ship> &ship)
 
 
 
+bool Ship::CanPickUp(const Flotsam &flotsam) const
+{
+	return this != flotsam.Source()
+		&& (government != flotsam.SourceGovernment() ||
+		(personality.Harvests() && !personality.IsAppeasing()))
+		&& cargo.Free() >= flotsam.UnitSize();
```

Comment:
Either put the operators at the beginning of each line or at the end (I prefer beginning), don't alternate.

Though, on consideration, I think something like the following would be even better:

```
bool Whatever(args)
{
    if(something)
        return false;

    if(something_else)
        return false;

    if(one_more_thing)
        return false;

    return true;
}
```

Reply:
Done.